hack/init-environment.sh

```diff
@@ -213,6 +213,45 @@ function install_secrets_manager () {
 }
 
 
+#####
+## Set up Google's CloudSQL Proxy to communicate with the CloudSQL database.
+#####
+function install-cloudsql-proxy () {
+  local -r kubeconfig=$1 env_dir=$2
```

Comment:
This should take one more arg for `env`, so we can parameterize the Vault read below.